scripts/run-operator.sh

```diff
@@ -15,9 +15,13 @@ then
   minikube start
 fi
 
-echo "building operator image in minikube"
+GIT_HASH="$(git rev-parse HEAD)"
+TMP_INSTALL_DIR="$(mktemp -d)"
+trap 'rm -rf -- "$TMP_INSTALL_DIR"' EXIT
+
+echo "building operator image in minikube for commit ${GIT_HASH}"
 KROXYLICIOUS_VERSION=${KROXYLICIOUS_VERSION:-$(mvn org.apache.maven.plugins:maven-help-plugin:3.4.0:evaluate -Dexpression=project.version -q -DforceStdout)}
-minikube image build . -f Dockerfile.operator -t quay.io/kroxylicious/operator:latest --build-opt=build-arg=KROXYLICIOUS_VERSION="${KROXYLICIOUS_VERSION}"
+minikube image build . -f Dockerfile.operator -t quay.io/kroxylicious/operator:${GIT_HASH} --build-opt=build-arg=KROXYLICIOUS_VERSION="${KROXYLICIOUS_VERSION}"
```

Comment:
what about extending the tag with `date "+%F-%T"` or similar e.g. 
```suggestion
minikube image build . -f Dockerfile.operator -t quay.io/kroxylicious/operator:${GIT_HASH}-$(date "+%F-%T") --build-opt=build-arg=KROXYLICIOUS_VERSION="${KROXYLICIOUS_VERSION}"
```
So its unique to the run.